[2022-06-17 09:07:29] [ERROR] gather cluster info error: Get "https://192.168.111.100:6443/version?timeout=32s": dial tcp 192.168.111.100:6443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchClusterInfo).Run.func1: 34) {map[]} 
[2022-06-17 09:07:59] [ERROR] gather cluster info error: Get "https://192.168.111.100:6443/version?timeout=32s": dial tcp 192.168.111.100:6443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchClusterInfo).Run.func1: 34) {map[]} 
